js_modules/dagit/packages/core/src/pipelines/__tests__/PipelineNav.test.tsx

```diff
@@ -0,0 +1,62 @@
+import {render, screen} from '@testing-library/react';
+import * as React from 'react';
+import {MemoryRouter} from 'react-router-dom';
+
+import {PipelineNav} from '../../nav/PipelineNav';
+import {TestPermissionsProvider} from '../../testing/TestPermissions';
+import {buildRepoAddress} from '../../workspace/buildRepoAddress';
+
+jest.mock('../../workspace/WorkspaceContext', () => ({
+  ...jest.requireActual('../../workspace/WorkspaceContext'),
+  useRepository: jest.fn(() => null),
+}));
+
+jest.mock('../../nav/JobMetadata', () => ({
+  JobMetadata: () => <div />,
+}));
+
+jest.mock('../../nav/RepositoryLink', () => ({
+  RepositoryLink: () => <div />,
+}));
+
+describe('PipelineNav', () => {
```

Comment:
Create this test to get some of the permissions checking that we were previously doing on PipelineRoot.test.